NETCORE/src/Microsoft.ApplicationInsights.AspNetCore/Extensions/ApplicationInsightsExtensions.cs

```diff
@@ -155,7 +155,7 @@ public static IServiceCollection AddApplicationInsightsTelemetry(this IServiceCo
 
                     // NetStandard2.0 has a package reference to Microsoft.Extensions.Logging.ApplicationInsights, and
                     // enables ApplicationInsightsLoggerProvider by default.
-#if NETSTANDARD2_0
+#if NETSTANDARD2_0 || NET461
```

Comment:
@SergeyKanzhelev, Today, this function is just called when we have a >.NET Standard 2.0. So, if we want to enable to .NET Framework, we should have to add .NET461 or more to be able to do that, so ILogger will be available as well. What do you think?